static/src/javascripts/bootstraps/common.js

```diff
@@ -238,18 +238,19 @@ define([
             // However, this means it's VITAL that all writes in callbacks are delegated to fastdom
             throttledScrollEvent: function () {
                 var running = false;
-                window.addEventListener('scroll', function () {
+                var emit = function () {
                     if (!running) {
                         running = true;
-                        // Use fastdom to avoid forced synchronous layout:
```

Comment:
Isn't this comment still true if we're not using RIC? (Look, I already created a cool new acronym for it)